languages/go/exercises/concept/comments/.docs/hints.md

```diff
@@ -0,0 +1,22 @@
+## General
+
+- [Documentation comments][comment] should be written directly before the thing that they are describing, start with the name of what they are describing, take the form of a sentence, and end with a period.
+
+## 1. Document the package
+
+- The [package comment][comment] should be written directly before the package, start with `Package X does Y.` and end with a period.
```

Comment:
We have the word `before` everywhere... maybe `above` explains better what is meant? Since it is about not having any empty lines in between...

Reply:
Most publishers I've worked with will correct 'above' to 'before' or 'previous' as part of their house style. The argument is something along the lines that it may be split over a page break, or whatever, in which case it's not 'above' any more, whereas 'before' is always correct. I think 'before' is probably better here too, even though the argument from books doesn't really apply.